config/gadi.config

```diff
@@ -1,38 +1,93 @@
 // profile for the Gadi supercomputer at NCI
-// Before running will need to run: 
-// module load nextflow singularity 
-
-// Gadi specific parameters
 // More info: https://opus.nci.org.au/display/DAE/Nextflow
-params {
-    pbs_account = ''
-    whoami = ''
-}
 
 // Preset use of containers with Singularity
 singularity {
     enabled = true
-    cacheDir = "/scratch/${params.pbs_account}/${params.whoami}/.nextflow/singularity"
+    cacheDir = "/scratch/er01/ndes8648/pipeline_work/nextflow/INFRA-83-Somatic-ShortV/singularity_cache"
     autoMounts = true
-    temp = "scratch/${params.pbs_account}/${params.whoami}/containers"
+    autoCleanUp = true
 }
 
 process {
-    module = 'singularity'
-    cache = 'lenient'
-    stageInMode = 'symlink'
-    executor = 'pbspro'
-    project = ''
-    storage = 'scratch/'
-    queue = 'normal'
-    
-    withName: 'processOne' {
+	//beforeScript = '${projectDir}/scripts/pullContainers.sh'
+	module = 'singularity'
+	cache = 'lenient'
+	stageInMode = 'symlink'
+	project = "${params.gadi_account}"
+	storage = "scratch/${params.gadi_account}+gdata/${params.gadi_account}"
+	disk = '30.GB'
+        
+
+withName: 'checkInputs' {
+	executor = 'local'
+}
+
+
+withName: 'mutect2' {
+        executor = 'pbspro'
+        queue = 'normal'
+        cpus = 1
+        time = '1h'
+        memory = '20.GB'
+}
+
+withName: 'GatherVcfs_step' {
+        executor = 'pbspro'
+        queue = 'normal'
+        cpus = 1
+        time = '1h'
+        memory = '20.GB'
+}
+
+withName: 'MergeMutectStats' {
+        executor = 'pbspro'
+        queue = 'normal'
+        cpus = 1
+        time = '1h'
+        memory = '20.GB'
+}
+
+withName: 'LearnReadOrientationModel' {
+        executor = 'pbspro'
+        queue = 'normal'
         cpus = 1
         time = '1h'
-        memory = '1GB'
+        memory = '20.GB'
 }
-    withName: 'processTwo' {
+
+withName: 'GetPileupSummaries' {
+        executor = 'pbspro'
+        queue = 'normal'
+        cpus = 1
+        time = '1h'
+        memory = '20.GB'
+}
+
+
+withName: 'CalculateContamination' {
+        executor = 'pbspro'
+        queue = 'normal'
+        cpus = 1
+        time = '1h'
+        memory = '20.GB'
+}
+
+withName: 'FilterMutectCalls' {
+        executor = 'pbspro'
+        queue = 'normal'
+        cpus = 1
+        time = '1h'
+        memory = '20.GB'
+}
+
+withName: 'getFilteredVariants_and_annotate' {
```

Comment:
How come this process has a combination of captialised first letters and underscores. Keep you naming conventions consistent throughout your codebase. Makes for easy reading by yourself and others.

Reply:
I have now made the names consistent. This particular file is not used . So deleted